call clearKeepAlive with ApiClient instead of websocket

cvium opened the pull request:
...

dmitrylyzo commented:
It works. But the comment states that there should be a websocket: `* @param {Object} instance The WebSocket connection.`
I believe `keepAliveInterval` should belong to the websocket, not `ApiClient`. At least that's how it was intended, imo.

cvium commented:
Maybe you're right. But we only have one socket anyway. I can change it if you want.

dmitrylyzo commented:
> Maybe you're right. But we only have one socket anyway. I can change it if you want.

Hmm, then it would be nice if you fix comments.

cvium commented:
> 
> 
> > Maybe you're right. But we only have one socket anyway. I can change it if you want.
> 
> Hmm, then it would be nice if you fix comments.

Renamed `instance` to `apiClient` and fixed the comments.
